src/cmake/build_libuhdr.cmake

```diff
@@ -0,0 +1,47 @@
+# Copyright Contributors to the OpenImageIO project.
+# SPDX-License-Identifier: Apache-2.0
+# https://github.com/AcademySoftwareFoundation/OpenImageIO
+
+######################################################################
+# libuhdr by hand!
+######################################################################
+
+set_cache (libuhdr_BUILD_VERSION 1.2.0 "libuhdr version for local builds")
+set (libuhdr_GIT_REPOSITORY "https://github.com/google/libultrahdr")
+set (libuhdr_GIT_TAG "v${libuhdr_BUILD_VERSION}")
+
+set_cache (libuhdr_BUILD_SHARED_LIBS OFF
+           DOC "Should execute a local libuhdr build, if necessary, build shared libraries" ADVANCED)
+
+build_dependency_with_cmake(libuhdr
+    VERSION         ${libuhdr_BUILD_VERSION}
+    GIT_REPOSITORY  ${libuhdr_GIT_REPOSITORY}
+    GIT_TAG         ${libuhdr_GIT_TAG}
+    CMAKE_ARGS
+        -D BUILD_SHARED_LIBS=${libuhdr_BUILD_SHARED_LIBS}
+        -D CMAKE_INSTALL_LIBDIR=lib
+        -D CMAKE_POSITION_INDEPENDENT_CODE=ON
+        -D UHDR_BUILD_EXAMPLES=FALSE
+        -D UHDR_ENABLE_LOGS=TRUE
+    )
+
+if (WIN32)
+    execute_process (COMMAND mkdir -p ${libuhdr_LOCAL_INSTALL_DIR}/lib)
+    execute_process (COMMAND cp ${libuhdr_LOCAL_BUILD_DIR}/Release/*.lib ${libuhdr_LOCAL_INSTALL_DIR}/lib)
```

Comment:
'cp' feels like it might not reliably work on Windows, no? Would it be more portable to use

    file (COPY ...)

or

    file (INSTALL ...)

??? (And that might even create the directory automatically, I'm not sure?

Reply:
You're right it's definitely a better alternative, code corrected !